ci-operator/config/openshift/lws-operator/openshift-lws-operator-release-4.19.yaml

```diff
@@ -37,7 +37,8 @@ tag_specification:
   namespace: ocp
 tests:
 - as: verify
-  commands: make verify-gofmt
+  commands: GOLANGCI_LINT_CACHE=/tmp/.cache GOCACHE=/tmp/ GOFLAGS='-mod=readonly'
```

Comment:
Adding linter in 4.19 is not required. But I'm cherrypicking all the changes into 4.19, in case we decide to release in 4.19. 

So that, I'll cherry-pick linter into 4.19, until that time this won't be functional.